src/app/server/Server.cpp

```diff
@@ -541,7 +541,11 @@ void Server::ResumeSubscriptions()
 Credentials::IgnoreCertificateValidityPeriodPolicy Server::sDefaultCertValidityPolicy;
 
 KvsPersistentStorageDelegate CommonCaseDeviceServerInitParams::sKvsPersistenStorageDelegate;
+#if CHIP_CRYPTO_PSA
```

Comment:
Is it possible to make some abstraction class that aggregates options for the keystone and uses here and below?

Reply:
We probably could but the extra work to do so is probably not worth it, especially given that we aren't going to be doing anything more with this going forward..